x-pack/docs/en/rest-api/security/saml-authenticate-api.asciidoc

```diff
@@ -0,0 +1,94 @@
+[role="xpack"]
+[[security-api-saml-authenticate]]
+=== SAML authenticate API
+
+Submits a SAML `Response` message to {es} for consumption.
+
+NOTE: This API is intended for use by custom web applications other than {kib}.
+If you are using {kib}, see the {stack-ov}/saml-guide.html[SAML guide].
```

Comment:
When this PR is rebased against master, you'll see that this content is all now in the elasticsearch repo for use in the Elasticsearch Reference, so the URLs should change as follows:
```suggestion
If you are using {kib}, see the <<saml-guide>>.
```